test/e2e/scaffold/scaffold.go

```diff
@@ -214,6 +214,26 @@ func (s *Scaffold) NewAPISIXClient() *httpexpect.Expect {
 	})
 }
 
+// NewAPISIXClient creates the default HTTP client.
```

Comment:
Pls ref: https://github.com/api7/api7-ingress-controller/blob/83873d0683dea5b1d0d78042d83c74374dbc4d4b/test/e2e/scaffold/scaffold.go#L428

Reply:
fixed. yeah i was using the wrong function.